foxpuppet/windows/browser/window.py

```diff
@@ -24,6 +25,17 @@ class BrowserWindow(BaseWindow):
     _tab_browser_locator = (By.ID, 'tabbrowser-tabs')
 
     @property
+    def navbar(self):
+        """Provides access to the Navigation Bar.
+        :returns: :py:class:`~foxpuppet.windows.browser.navbar.NavBar`
+        :return type: object
+        """
+        with self.selenium.context(self.selenium.CONTEXT_CHROME):
+            navbar = self.selenium.find_element(*self._nav_bar_locator)
+        window = BaseWindow(self.selenium, self.selenium.current_window_handle)
+        return NavBar(window, navbar)
```

Comment:
I suspect it makes no difference, but I think this would read better as:

```python
window = BaseWindow(self.selenium, self.selenium.current_window_handle)
with self.selenium.context(self.selenium.CONTEXT_CHROME):
    el = self.selenium.find_element(*self._nav_bar_locator)
    return NavBar(window, el)
```

What do you think?